kuka_rsi_driver/launch/dual_arm_startup.launch.py

```
@@ -0,0 +1,414 @@
+    }
+
+    controller_spawners = [
+        controller_spawner(name, prefix_cmd, param_file)
+        for name, param_file in controllers.items()
+    ]
+
+    nodes_to_start = [
+        control_node,
+        robot_manager_node,
+        robot_state_publisher,
+    ] + controller_spawners + [cleanup_event_broadcaster_config_on_shutdown]
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)